lib/server/generate.js

```diff
@@ -41,15 +41,12 @@ function execute() {
   // takes the content of a doc article and returns the content with a table of
   // contents inserted
   const insertTableOfContents = rawContent => {
-    const regexp = /\n###\s+(`.*`.*)\n/g;
-    let match;
-    const headers = [];
-    while ((match = regexp.exec(rawContent))) {
-      headers.push(match[1]);
-    }
+    const filterRe = /`[^`]*`/;
```

Comment:
Anchor this with both `^` and `$` so it only matches when the backticks wrap the full heading like the old regexp.

Reply:
Ok. I'll do that, but the old one can match the following:

```
###     `foo` bar baz
```

This is why I've decided, that the match should not be strict

Reply:
Only anchor the beginning then. Trying to infer the intention of the old one.